src/containers/OasisTrade.test.js

```diff
@@ -1,22 +1,275 @@
 /* global shallow describe it expect */
-/* eslint-disable import/first */
+/* eslint-disable import/first,no-undef */
 import React from 'react';
 import Immutable from 'immutable';
+import { shallow } from 'enzyme';
+
 
 import {
-  OasisTradeWrapper,
-  mapStateToProps,
-  mapDispatchToProps,
+OasisTradeWrapper,
+mapStateToProps,
+mapDispatchToProps,
 } from './OasisTrade';
-import { shallow } from 'enzyme';
 
 describe('(Container) OasisTrade', () => {
-  const state = Immutable.fromJS({});
+  const state = Immutable.fromJS({
+    tokens: {
+      allTokens: [
+        'W-ETH',
+        'MKR',
+        'DGD',
+        'GNT',
+        'W-GNT',
+        'REP',
+        'ICN',
+        '1ST',
+        'SNGLS',
+        'VSL',
+        'PLU',
+        'MLN',
+        'RHOC',
+        'TIME',
+        'GUP',
+        'BAT',
+        'NMR',
+        'SAI',
+      ],
+      baseTokens: [
```

Comment:
Probably you can get those from the constants you have, otherwise what was the points of the constants